userspace/libscap/engine/gvisor/CMakeLists.txt

```diff
@@ -56,15 +56,11 @@ add_custom_command(
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 )
 
-if(USE_BUNDLED_JSONCPP)
-	add_dependencies(scap jsoncpp)
-endif()
-
 add_library(
 	scap_engine_gvisor ${scap_engine_gvisor_sources} ${scap_engine_gvisor_generated_sources}
 )
 
-add_dependencies(scap_engine_gvisor uthash)
+add_dependencies(scap_engine_gvisor uthash jsoncpp)
```

Comment:
No need to 
```
if(USE_BUNDLED_JSONCPP)
```
since the jsoncpp cmake module always creates the `jsoncpp` target.